cmd/exporters/prometheus/prometheus.go

```diff
@@ -97,17 +97,17 @@ func (me *Prometheus) Init() error {
 	}
 
 	// add HELP and TYPE tags to exported metrics if requested
-	if me.Params.GetChildContentS("add_meta_tags") == "true" {
+	if me.Params.ShouldAddMetaTags != nil && *me.Params.ShouldAddMetaTags == true {
```

Comment:
clearer to say
if me.Params.ShouldAddMetaTags != nil && *me.Params.ShouldAddMetaTags {